src/ProjectTemplates/test/Infrastructure/TemplateTests.props.in

```diff
@@ -16,22 +16,12 @@
 
     <KnownFrameworkReference
       Update="Microsoft.AspNetCore.App"
-      TargetFramework="${DefaultNetCoreTargetFramework}"
       DefaultRuntimeFrameworkVersion="${MicrosoftAspNetCoreAppRuntimePackageVersion}"
       LatestRuntimeFrameworkVersion="${MicrosoftAspNetCoreAppRuntimePackageVersion}"
       TargetingPackVersion="${MicrosoftAspNetCoreAppRefPackageVersion}"
       RuntimePackRuntimeIdentifiers="${SupportedRuntimeIdentifiers}" />
```

Comment:
Should this look more like the `@(KnownFrameworkReference)` changes done in eng/Workarounds.targets and related files? Main issue is this updates the Microsoft.AspNetCore.App reference for all TFMs to use the latest ref and runtime packages. While this might be fine now, it could cause problems when "new" TFM isn't recognized in the SDK yet or if we have any tests using downlevel TFMs.

Reply:
Ideally yes, but that hits the issue from https://github.com/aspnet/AspNetCore/pull/14670#discussion_r331323554, where msbuild complains about conditions on item metadata here. Will open an issue on this after I merge, since it will take some investigation to figure out why that's happening only here.